src/parse.ts

```diff
@@ -85,10 +85,13 @@ export function parse(messageDefinition: string, options: ParseOptions = {}): Me
 }
 
 export function fixupTypes(types: MessageDefinition[]): void {
```

Comment:
could we add a comment to describe what this function is supposed to be doing?

Reply:
Also, I see that there is a `skipTypeFixup` option which skips calling this function. Do we still need to do anything to fix namespace-confusion bugs if `skipTypeFixup: true` is used?

Reply:
> Like [here](https://github.com/foxglove/rosmsg/blob/5cd65f9092b6b8b6783b5a92678efbeea5297928/src/gendeps2.ts#L83)? but maybe it is already handled correctly?

Looks like the lines below that referenced line are doing more or less the same as what `fixupTypes` does. https://github.com/foxglove/rosmsg/blob/5cd65f9092b6b8b6783b5a92678efbeea5297928/src/gendeps2.ts#L84-L89